src/test/ut/ut_cortxfs_io_ops.c

```diff
@@ -182,6 +182,46 @@ static void test_rw_4k(void **state)
 	free(buf_out);
 }
 
+/**
+ * This test will validat the scenario where we tried to read from EOF
```

Comment:
validat -> validate

Reply:
Addressed at 03c6f6476b07ba88003edde772255433877af479

Reply:
Resolving comments as @sachinpunadikar  has approved the changes